lib/config/fileconf.go

```diff
@@ -512,8 +512,10 @@ type ConnectionRate struct {
 // ConnectionLimits sets up connection limiter
 type ConnectionLimits struct {
 	MaxConnections int64            `yaml:"max_connections"`
-	MaxUsers       int              `yaml:"max_users"`
 	Rates          []ConnectionRate `yaml:"rates,omitempty"`
+
+	// MaxUsers is deprecated and has no effect.
+	MaxUsers int `yaml:"max_users"`
```

Comment:
IIUC this PR deprecates the field max_users field? How will we communicate this to users? v17 changelog entry?

It would be nice to add a changelog entry on the PR, unless you're planning to do it somewhere else.

Reply:
The max users field never actually limited the number of max users, so there isn't actually a behavior change here. See the description in the commit message for the juicy details.